cli/src/config.rs

```diff
@@ -195,6 +194,18 @@ pub fn get_home_settings_path() -> Result<PathBuf> {
     Ok(config_path)
 }
 
+/// Resolve XDG data directory.
+pub fn data_dir() -> Result<PathBuf> {
+    let home_path =
```

Comment:
If for some reason we can't find the user's home directory, this code bails out even if it isn't going to use the home directory.